content/articles/2018-08-05-google-summer-of-code-2018.md

```diff
@@ -0,0 +1,24 @@
+2. [First milestone, first part](https://medium.com/@nikita.astronaut/the-end-of-may-in-gsoc18-poliastro-project-2d3ed2d6c9e0). Wrote my own DOP853 solver in python!! [here](https://github.com/poliastro/poliastro/pull/368) and optimized orbit.sample method using now accessible dense output [here](https://github.com/poliastro/poliastro/pull/379).
+3. [First milestone, second part](https://medium.com/@nikita.astronaut/the-detective-story-3rd-body-perturbation-validation-8b260086ea0d). After a veeery long search for mistake in validation (it actually turned out to be mistake in BOOK), added 3rd-body perturbation [here](https://github.com/poliastro/poliastro/pull/381).
+4. [Second milestone, first part](https://medium.com/@nikita.astronaut/solar-drag-and-robust-kepler-equation-solution-694a4658c35b). Added solar drag perturbation [here](https://github.com/poliastro/poliastro/pull/388), changed the source of docs download [here](https://github.com/poliastro/poliastro/pull/390) and implemented better treatment of parabolic region in mean_motion propagator [here](https://github.com/poliastro/poliastro/pull/394).
+5. [Second milestone, second part](https://medium.com/@nikita.astronaut/ending-of-2nd-evaluation-period-new-propagator-in-parabolic-region-j3-perturbation-continuous-17042f85d8b3). Added J3-perturbations [here](https://github.com/poliastro/poliastro/pull/398) and artificial perturbations from the old Juan's code [here](https://github.com/poliastro/poliastro/pull/400).
+6. [Third milestone](https://medium.com/@nikita.astronaut/making-propagators-fast-again-and-more-a8df832ca7ec). Made propagators fasted by using numba [here](https://github.com/poliastro/poliastro/pull/409) and [there](https://github.com/poliastro/poliastro/pull/413), added continuous benchmarking of propagators [here](https://github.com/poliastro/poliastro-benchmarks/pull/16) and spent some time documenting and demonstrating new features from this summmer [here](https://github.com/poliastro/poliastro/pull/428).
+
+## Conclusion
+So, as you can see, quite some work has been done and I got really tired :) However, I want to thank Juan for his continuous assessment and support, for teaching me right code structuring and using github. Juan was always ready to help and save me from any bugs, which made all these contributions possible. This was a real experience and I am happy that I agreed to take part in it :)
```

Comment:
Do you feel like adding a couple of screenshots from https://docs.poliastro.space/en/latest/examples/Natural%20and%20artificial%20perturbations.html? :)